### ISSUE TITLE
Issuers are not forced to be fetched after connecting to a custodian

### ISSUE THREAD

**Issue by btlechowski** [brave/brave-browser#33207]
Follow up to https://github.com/brave/brave-browser/issues/31769. Issue found by @GeetaSarvadnya 
It seems the issuers are only downloaded when scheduled beforehand
```
[6692:6692:0922/163923.883907:VERBOSE1:issuers_url_request.cc(129)] Fetch issuers in 0 hours, 29 minutes, 33 seconds at 17:08:56.882
```

Note: issuers are eventually fetched

Important: when verifying the issue make sure that re-fetching issuers is forced and not from being scheduled beforehand.

## Steps to Reproduce

1. Clean profile
2. Enable rewards
3. Connect to a custodian
4. Check logs if fetching of the issuers is forced

## Actual result:

Re-fetching of the issuers is not forced. 

Issuers are fetched when scheduled:
```
[6692:6692:0922/163923.883907:VERBOSE1:issuers_url_request.cc(129)] Fetch issuers in 0 hours, 29 minutes, 33 seconds at 17:08:56.882
```

## Expected result:
Re-fetching of the issuers is forced. 
Issuers are downloaded just after connecting to the custodian

## Reproduces how often: 

Easily reproduced

## Brave version (brave://version info)

Brave | 1.58.131 Chromium: 117.0.5938.92 (Official Build) (64-bit)
-- | --
Revision | 2cd98952c615ae83a524a0285aa7dcf5acac5aea
OS | Linux

cc @tmancey @aseren

**Comment by tmancey** [comment]
@btlechowski this is working as expected. Because we only follow those use cases if the user previously had a connected wallet before upgrading to Rewards 2.5.
```
bool StateMigrationV13::MigrateExternalWallet(const std::string& wallet_type) {
  if (!wallet::GetWalletIf(*engine_, wallet_type,
                           {mojom::WalletStatus::kConnected})) {
    BLOG(1, "User doesn't have a connected " << wallet_type << " wallet.");
  } else {
    engine_->client()->ExternalWalletConnected();
  }

  return true;
}
```

**Comment by btlechowski** [comment]
Verified with

Brave | 1.60.87 Chromium: 118.0.5993.70 (Official Build) beta (64-bit)
-- | --
Revision | 854c1e600054dfca6842cebaf729f100f4b0f066
OS | Linux

Verified test plan from the description

Verified issuers are downloaded after connecting to a custodian

```
18328:18328:1012/184833.041877:VERBOSE1:account.cc(273)] Shutdown issuers url request
[18328:18328:1012/184833.044579:VERBOSE1:account.cc(276)] Reset issuers
[18328:18328:1012/184833.044830:VERBOSE1:account.cc(235)] Initialize issuers url request
[18328:18328:1012/184833.044999:VERBOSE1:issuers_url_request.cc(61)] Fetch issuers
[18328:18328:1012/184833.045213:VERBOSE6:issuers_url_request.cc(67)] URL Request:
  URL: https://static.ads.bravesoftware.com/v3/issuers
  Method: kGet
[18328:18328:1012/184833.045976:VERBOSE7:issuers_url_request.cc(68)]   Headers:

[18328:18328:1012/184833.072546:VERBOSE1:notification_ad_event_handler.cc(36)] Failed to fire notification ad event due to missing placement id e64373ac-2ca5-4f6b-b497-1f1d7ccd40c8
[18328:18328:1012/184833.078887:VERBOSE1:notification_ad_event_handler.cc(79)] Failed to fire notification ad kServed event for placement id e64373ac-2ca5-4f6b-b497-1f1d7ccd40c8
[18328:18328:1012/184833.079530:VERBOSE9:confirmation_state_manager.cc(189)] Successfully saved confirmation state
[18328:18328:1012/184833.081201:VERBOSE9:confirmation_state_manager.cc(189)] Successfully saved confirmation state
[18328:18328:1012/184833.085420:VERBOSE9:confirmation_state_manager.cc(189)] Successfully saved confirmation state
[18328:18328:1012/184833.157676:VERBOSE5:rewards_engine_impl.h(260)] 
[ REQUEST ]
> URL: https://api-sandbox.uphold.com/v0/me/cards/fbd1546c-6706-4267-9e6c-3eda44945283
> Method: GET
[18328:18328:1012/184833.221659:VERBOSE5:rewards_engine_impl.h(260)] 
[ REQUEST ]
> URL: https://api-sandbox.uphold.com/v0/me/cards/fbd1546c-6706-4267-9e6c-3eda44945283
> Method: GET
[18328:18328:1012/184833.248693:VERBOSE6:issuers_url_request.cc(77)] URL Response:
  URL: https://static.ads.bravesoftware.com/v3/issuers
  Response Status Code: 200
  Response: {"ping":7200000,"issuers":[{"name":"payments","publicKeys":[{"publicKey":"InLY6FmxKtSETLo6iY8QkNfFQ+AAyLfT1K+aCqsR7BI=","associatedValue":"0.3"},{"publicKey":"kAzm4HCyTj6daC934iVNI+1gsLZT/xJC7arWRK3sFQE=","associatedValue":"0.3"},{"publicKey":"tECg8xW+B9Fpx7J+GDc+yhSLScSjmjSZolHfoSMRBEs=","associatedValue":"0.3"},{"publicKey":"aMhP+AG47JlAT1QDBRXeS/jWejzXCASYaMx70ALcjQQ=","associatedValue":"0.25"},{"publicKey":"bnlAqCCEE0NlfQgW5Ak0AIeKKw/nHIEdvFlPckvTCXk=","associatedValue":"0.25"},{"publicKey":"Qu4VjcrqgQoRm01WmNifDv/XPp9020pdDCau+PcQMXI=","associatedValue":"0.25"},{"publicKey":"NmwXEbeIi1q95VSG6JavHwK3baaJVt6xVYFMVfVl+zI=","associatedValue":"0.20"},{"publicKey":"cBLCjab8oZ2CiEgvxd2lrQ9TtIKAPUcAIDsxYFd7sxM=","associatedValue":"0.20"},{"publicKey":"/kSZCTUqM47WwMOZ3kq+y391uFctT89p4YmzPkt/kxA=","associatedValue":"0.20"},{"publicKey":"Dhl85HrxxU5cAwFByh2iD3qoqbE//aK0CLpLgAaT5S4=","associatedValue":"0.15"},{"publicKey":"YBaB4yjyReCbDy2Vi498jGpyN07poAXP30kfJImhwR0=","associatedValue":"0.15"},{"publicKey":"qIx4qpbq6iAGplZaR5y+JhSGxOWYmJQTjTc+vng4JWU=","associatedValue":"0.15"},{"publicKey":"cmSUuF1wgOyZNXZ9brM03cTah9nL7BGHxSp5JpBM3Bk=","associatedValue":"0.10"},{"publicKey":"uDiPcPAAJGZT3jgdSBkTIA3rOk4rQrBM8DCmPxYMM3c=","associatedValue":"0.10"},{"publicKey":"ZO9kLo2RmOt9tkYwEW7OSuT9lvFu6nMOxp5ty/c+h2A=","associatedValue":"0.10"},{"publicKey":"urDejI9ebt9McXLW7Ackr07B4AjUOXnLQOV3wThFwQs=","associatedValue":"0.05"},{"publicKey":"Vl4jtWAxP3gZ5oZAThbuAXhzwV9f4rQJmSfQ+O5eJRU=","associatedValue":"0.05"},{"publicKey":"MH69A4AAxzE97sTr28h3WunlcCqFw6XdTm6b2nNR/04=","associatedValue":"0.05"},{"publicKey":"JjGdPs7ilnZlM4yyuhZFx+eHyPz5A38HiAwCQOOoogA=","associatedValue":"0.025"},{"publicKey":"7t7rwEN2fQ0PaH2W00GEE303x8epN2nqA2JPq8gPoF8=","associatedValue":"0.025"},{"publicKey":"ULuc2wjogn36+H+i8DgnsoBOXz2N0fp3FtZ5oG2Ij0k=","associatedValue":"0.025"},{"publicKey":"qiy6l/i2WXc8AkDFt/QDxXoT0XKgL0xRSZ3Db+k2u3A=","associatedValue":"0.01"},{"publicKey":"BF4cfMiT9ph89IO1oZf7wrKQeX0YW/Hq2T1CyriXc0E=","associatedValue":"0.01"},{"publicKey":"CiHciXdOBJul3B24M7JuumGSsGcfOK22TGapg8ybqDY=","associatedValue":"0.01"},{"publicKey":"gOX3HSuB/CHNouVbHnEGi26KSTDPOVdO0ntyO5RnT3A=","associatedValue":"0"},{"publicKey":"YBKv6tPgXkngMQL6pw/3ZierxOM5M4tp46iQfAw05QE=","associatedValue":"0"},{"publicKey":"ROaQZ+zBlQAtBqRKOrEIZNvZ+FuwwxHSblhUzG6QNzo=","associatedValue":"0"},{"publicKey":"ah5CZEmBtN6CTEnL9UJEGa+PNf5IMVNhxpDuP306aTo=","associatedValue":"0.10"},{"publicKey":"+vQV6eMTJHaAvLAun3VYZVMJ1Sn/nf/BlnWczIRLBh4=","associatedValue":"0.05"},{"publicKey":"dmdCB00i/VUzaWUOGfBokgVgQjLq9BIMx1glwtqYcBI=","associatedValue":"0.025"},{"publicKey":"gN4kY8+sw1vJ8NZ2tl8FsTM9Vwu8kfMjzbJLQaTXYkA=","associatedValue":"0.01"},{"publicKey":"7K5RWYAjgFEIWmESDQ3ZHb4ScU3wGo0gNQ1Lp+USvj4=","associatedValue":"0.025"},{"publicKey":"5iBxAKt7LdcrA4ZN+07h8Cu3QuQwzg0p55RsGi3nMXc=","associatedValue":"0"},{"publicKey":"+H3+zRKCpEXlR/5vnziJcd+m6h0yrIty1Xjpc3C7HA4=","associatedValue":"0.3"},{"publicKey":"0PSKczpUxiiHmu+IMfqlH1/GU4DsZOLLb5wK2bxFwCw=","associatedValue":"0.10"},{"publicKey":"hsMZrVyq+OjyrhKeY6fpzXr6iwVkDKhMRbiLOliXRWE=","associatedValue":"0.15"},{"publicKey":"yldP7BSs0sGGoDEv1JN7SYYQk51JjuLSXowbTY+jXnQ=","associatedValue":"0.20"},{"publicKey":"5n0dX/kfS5E+xt0Mu2vKSL79snYZeQBjLATQjgzTbyw=","associatedValue":"0.25"},{"publicKey":"FgwCB+zb+prkgXdP8CGS5AnTrt4ZP5nwVze5d3OvlmM=","associatedValue":"0.3"},{"publicKey":"WHqbHLUCki23/7ZteaVrSJShC0uwtzPn0cVoPrVZkxM=","associatedValue":"0"},{"publicKey":"jlzfhkDPZ2xWcg3zRZ2QcvcPTWUwXdTm/gvaTzcKXyc=","associatedValue":"0.05"},{"publicKey":"sBsRdtb9iAyl8sUGPED57oo09/DEnQrcE5iNssAaJk0=","associatedValue":"0.01"},{"publicKey":"ehMwVjrw7Ih1YL6TVLtlPUzIpUaKRnPsIdVEtxvrJ1k=","associatedValue":"0.25"},{"publicKey":"+Nm4QCZN2U7wY5Cs27pIAEsJtRd9ZipU36JkSOtql3E=","associatedValue":"0.20"},{"publicKey":"pNYHIB7MNWL6TXiJ0tHjt7h4w52tcNKDT8rwQl6UMS4=","associatedValue":"0.15"}]},{"name":"confirmations","publicKeys":[{"publicKey":"hLaC3RQw5++hx8zaT+47CA8r8cOlAm+ygeky/ci6T1s=","associatedValue":""},{"publicKey":"WGeOYxJNSG0Mn/BbNwNbKPxrbFCannzM8QvP2z0zDwU=","associatedValue":""},{"publicKey":"Ptf1If56suOe3TR9BrTnDAhMYS3b54T+Z2mCieZGNEo=","associatedValue":""},{"publicKey":"wgMhKwj/Vs832efOg+RlqJvLy3Ta3UDbz1wV+Z+dzEU=","associatedValue":""},{"publicKey":"hgNa/G7BVyIpjG0p9/go+iOPKvPto4ZRVChGEbf8aHk=","associatedValue":""},{"publicKey":"YKcTSIjDgGdBAZSvWZY639n4Duv6xpcpmMTXyPgxRDo=","associatedValue":""}]}]}
[18328:18328:1012/184833.251984:VERBOSE7:issuers_url_request.cc(78)]   Headers:
    access-control-allow-credentials: true
    content-length: 4450
    content-security-policy: sandbox
    content-type: application/json; charset=utf-8
    cross-origin-opener-policy: same-origin
    cross-origin-resource-policy: same-origin
    date: Thu, 12 Oct 2023 17:04:16 GMT
    referrer-policy: same-origin
    strict-transport-security: max-age=31536000; includeSubDomains; preload
    via: 1.1 bdb480ba487636e194d63f984ed846f2.cloudfront.net (CloudFront)
    x-amz-cf-id: cdDLfeb-35l4qtBmScFrZFhUAaMs82drNwkgH0IvBh175dwQHLb7GQ==
    x-amz-cf-pop: TXL50-P1
    x-cache: Miss from cloudfront
    x-content-type-options: nosniff
    x-frame-options: DENY
[18328:18328:1012/184833.255695:VERBOSE1:issuers_url_request.cc(86)] Parsing issuers
[18328:18328:1012/184833.256131:VERBOSE1:issuers_url_request.cc(100)] Successfully fetched issuers
[18328:18328:1012/184833.258639:VERBOSE1:account.cc(51)] Updated issuers
```


Verified confirmation tokens are downloaded after downloading issuers
```
[18328:18328:1012/184833.265583:VERBOSE1:account.cc(436)] Refill confirmation tokens
[18328:18328:1012/184833.266162:VERBOSE1:refill_confirmation_tokens.cc(102)] Request signed tokens
[18328:18328:1012/184833.266868:VERBOSE6:refill_confirmation_tokens.cc(107)] URL Request:
  URL: https://mywallet.ads.bravesoftware.com/v3/confirmation/token/5f1220cd-70f9-523c-a658-f5a0074bbb0b
  Content: {"blindedTokens":["qKMWXHxoutdKRDCJ6W14RDbqWcnDxNKPcZKPRUNlk08=","eve/GnvrXnnKCRvXYfXs5MgT/lyo2sGuXrx+MV4arz8=","HF67Uw/kxqfVk1nX8h3oaJCWKZJOIfLFwS3fP1o+Uhc=","bDwemNmxPTBdN5fJdvon1HfM8c2Uxs68E+406Slqmkc=","FiDABdVXMfPGYZj+7DGbbEcFGtXKTFyUaOSyxyNDRgY=","oB7AQafm+/wcJVOyUvaul4WEHw38cm8gdqHHnHqxAlo=","5qRiUMsMgOckkJf6CgzvHl3crLt3kL3fcF05AcxBeV0=","dqMEqTfIdGsmP/1c9uTEKXnisw83h2SnL3EEPFNQ03s=","nFug+lgWxpg6t8JTsJWF4AMRDaNg+bb1/8wOY5DlKnE=","RlnzvJqdh3Jd5l3PP4wUXXfuIMIDd0tNZ7vp1tt5JFI=","ytT7hm6c9tb6YmnWl55DD1cwIhQ1K5/aUiUTmi/FbQs=","2iyf09gisYx4bL+zFJRHxt9GgtzYK3tPinrCs006ZA8=","XEK9Z7uD0bE04gP6N0aM/NUxNYkmXNMuKg+9st6zlko=","7tFUpL1fD3yojdeZdpN6kbpsNvOaztc+OcfbHSC0XkI=","VjhX/CmdwyJtZ4o1WwYrkPkHlvXY4gNn95wGuFrUTwA=","0qra3lmz7/eYnR/ydr8TKfR/pwS5oQCAraNJyMHrdzQ=","Yk+XrdhCL2kFqCXcdCh0nCVwVpLpP7mqODWakUy/VxQ=","6jDaxTBmc/S4Vr6JV8TgLvUtkBAELJqQ0tB7aL3UCFU=","DHl4rTAgRyzKyF2JBaKsCrgD1g4BXB0whu9w0VMpNUw=","Uhu5JI4vS4Ck3Xs4B0EsMEFyiFZR1m0KhXISzcCKZXg=","dJIFZ3rkgkIKq+ttGgbPFrb9Tt9NoEhcAscA7jQDOjo=","1ng73Vx2PQU7Wpabo0UU6v9g/cq7pyVkxXSBKkAdkiE=","zASDKk1PVCkbtuZSsPFbJeMJWrab7KqBlsiaHt4MkWQ=","uowZ8/w631V1B/Fx+6nSYm+aaQpMAOKeCCdOr8Mm7Rc=","6s4D7zB5DY6wk/gHjBPcsdocQdRM8YJCz592RkgVMiY=","uDQpPYzO/c1wRQSCSJwrlCIEMDMP6jqSSwUpQMcVdDQ=","VFJuY45OBxpAeN9t50EnXturvnbas0BrlDbWz8HHUxg=","ZoZ6aMmOZgXZ1ehKyfOQsEjLD8b+T6ratQvYX+CAyEM=","IOTboANYQIzgURxxRahioc+zcklUCg/EEyvOSyM3XhQ=","pvhiQdWK0BOL3U/knzCC9xjkYLU+JOkKh6bnRP20Jwg=","EBJ6QoWZm2an+iI79Z7Wyp/bGiHo5TQ7ebc05y8Glk0=","uG6QbDu9VVy06j+t8cl7SFKmO/ymQmHCE+ic8hlfSVI=","cP26YVuhb+npuNv38TM+fVp07cBfZ3brrz8hkD8FbFA=","8vHEii6sU0HoboaR+yBC3Jz4ReQ7glcayzNaBelTgnY=","YOmZSInkcsvPkBx/ISfQkmQwxt20WWu+AuPdwk9GZjw=","XBs3INeuTHpbDUVng892iMfFQ06q/Rin3rP8MsJSy0o=","qtnluzPCy1lnBS17S8qgXdqvNRrMGboIOn9x4bgNrF0=","aiFvZXjawnUKCEe+GorYMRWCaaTzC77F4lxreFoJI0Y=","AHENAAIZzOfHnbXx5TYm0zJ4U46+ClapMQAovsl3Hg4=","VvwD3rQSIm2qN9nS+kSXH/ZxgI7uIGVjZc28kjnKcmk=","RKCirOUKqziKgLOYmJNRzvp0Y786UY/Spl8JxbkV4D8=","etH8/YAPlVgrzsoOepEDDJUdpYIuQOv221txpsv48Gk=","0I0ox+h1gxDn9jwR2zsIakrkVkn0OCMYiL1LAOhTPnI=","jA8/VmAogvtQ5+6RDHe/qKPD/RFVztA5omtC0x09dTI=","DDIzx2tUalw1DuQihS0C3lbujKqgAD/myAx4ZICOaXc=","qpoSt3xzsSVE5ZByUt9WIYuJsQOnq7YGWgPXc/TZyGg=","APhuxv0U0wsyya6c2jnqZTlyysKedKroa8R1DruT5mc=","Fq9XU6VwFwOISzk+L1JTlxrOt9Gkv5k3xwdgl7I11zI=","zqfaOeo6B2iH6XJ3tJL4AWP9/zHb1XGt7ZuFi8mvIzI=","NJCoecWR2Be6NGUwHzPj78TZhj7VF2txl+vYy0ENh1g="]}
  Content Type: application/json
  Method: kPost
[18328:18328:1012/184833.267030:VERBOSE7:refill_confirmation_tokens.cc(108)]   Headers:
    digest: SHA-256=N6hBhyUyvs6kC0Au2P0M457dzFVyeC8zB7gpkRzbWjs=
    signature: keyId="primary",algorithm="ed25519",headers="digest",signature="5cJN3HoFvGT7VUbMuF52gSpw3hwO2lLUP63JXhCt4L+f9utE93Y876gTINs3USaJv90XztBW4V449sNPgM7eDQ=="
    content-type: application/json
    accept: application/json
[18328:18328:1012/184833.269577:VERBOSE1:issuers_url_request.cc(122)] Fetch issuers in 4 hours, 17 minutes, 1 second at 23:05:34.267
[18328:18328:1012/184833.358896:VERBOSE6:logging_util.cc(120)] 
[ RESPONSE - OnRequest ]
> Url: https://api-sandbox.uphold.com/v0/me/cards/fbd1546c-6706-4267-9e6c-3eda44945283
> Result: Success
> HTTP Code: 200
> Body: {"CreatedByApplicationId":"193a77cf-02e8-4e10-8127-8a1b5a8bfece","address":{"ethereum":"0x2389D6cCA77f595C87F6542BE097515932570C51","wire":"UH3C41037A"},"available":"10.1","balance":"10.1","currency":"BAT","id":"fbd1546c-6706-4267-9e6c-3eda44945283","label":"Brave Browser","lastTransactionAt":"2023-02-23T15:15:59.602Z","normalized":[{"available":"1.60","balance":"1.60","currency":"EUR"}],"settings":{"position":1,"protected":false,"starred":false},"wire":[{"accountName":"Uphold Europe Limited","address":{"line1":"Tartu mnt 2","line2":"10145 Tallinn, Estonia"},"bic":"LHVBEE22","currency":"EUR","iban":"EE76 7700 7710 0159 0178","name":"AS LHV Pank"},{"accountName":"Uphold Europe Limited","accountNumber":"00000083","address":{"line1":"1 Angel Court","line2":"London, United Kingdom, EC2R 7HJ"},"bic":"LHVBEE22","currency":"GBP","iban":"EE76 7700 7710 0159 0178","name":"LHV BANK LIMITED","sortCode":"040300"}]}
[18328:18328:1012/184833.507359:VERBOSE6:logging_util.cc(120)] 
[ RESPONSE - OnRequest ]
> Url: https://api-sandbox.uphold.com/v0/me/cards/fbd1546c-6706-4267-9e6c-3eda44945283
> Result: Success
> HTTP Code: 200
> Body: {"CreatedByApplicationId":"193a77cf-02e8-4e10-8127-8a1b5a8bfece","address":{"ethereum":"0x2389D6cCA77f595C87F6542BE097515932570C51","wire":"UH3C41037A"},"available":"10.1","balance":"10.1","currency":"BAT","id":"fbd1546c-6706-4267-9e6c-3eda44945283","label":"Brave Browser","lastTransactionAt":"2023-02-23T15:15:59.602Z","normalized":[{"available":"1.60","balance":"1.60","currency":"EUR"}],"settings":{"position":1,"protected":false,"starred":false},"wire":[{"accountName":"Uphold Europe Limited","address":{"line1":"Tartu mnt 2","line2":"10145 Tallinn, Estonia"},"bic":"LHVBEE22","currency":"EUR","iban":"EE76 7700 7710 0159 0178","name":"AS LHV Pank"},{"accountName":"Uphold Europe Limited","accountNumber":"00000083","address":{"line1":"1 Angel Court","line2":"London, United Kingdom, EC2R 7HJ"},"bic":"LHVBEE22","currency":"GBP","iban":"EE76 7700 7710 0159 0178","name":"LHV BANK LIMITED","sortCode":"040300"}]}
[18328:18328:1012/184833.646662:VERBOSE1:user_idle_detection.cc(46)] User is idle
[18328:18328:1012/184833.753942:VERBOSE6:refill_confirmation_tokens.cc(118)] URL Response:
  URL: https://mywallet.ads.bravesoftware.com/v3/confirmation/token/5f1220cd-70f9-523c-a658-f5a0074bbb0b
  Response Status Code: 201
  Response: {"nonce":"1e8b54d3-a601-499d-8444-3f629583de43"}
[18328:18328:1012/184833.754077:VERBOSE7:refill_confirmation_tokens.cc(119)]   Headers:
    accept-ranges: bytes
    access-control-allow-credentials: true
    content-length: 48
    content-security-policy: sandbox
    content-type: application/json; charset=utf-8
    cross-origin-opener-policy: same-origin
    cross-origin-resource-policy: same-origin
    date: Thu, 12 Oct 2023 17:04:17 GMT
    referrer-policy: same-origin
    strict-transport-security: max-age=31536000; includeSubDomains; preload
    vary: Origin
    via: 1.1 varnish
    x-cache: MISS
    x-cache-hits: 0
    x-content-type-options: nosniff
    x-frame-options: DENY
    x-served-by: cache-fra-eddf8230040-FRA
[18328:18328:1012/184833.754152:VERBOSE1:refill_confirmation_tokens.cc(166)] Get signed tokens
[18328:18328:1012/184833.754880:VERBOSE6:refill_confirmation_tokens.cc(170)] URL Request:
  URL: https://mywallet.ads.bravesoftware.com/v3/confirmation/token/5f1220cd-70f9-523c-a658-f5a0074bbb0b?nonce=1e8b54d3-a601-499d-8444-3f629583de43
  Method: kGet
[18328:18328:1012/184833.755052:VERBOSE7:refill_confirmation_tokens.cc(171)]   Headers:

[18328:18328:1012/184833.956121:VERBOSE6:refill_confirmation_tokens.cc(181)] URL Response:
  URL: https://mywallet.ads.bravesoftware.com/v3/confirmation/token/5f1220cd-70f9-523c-a658-f5a0074bbb0b?nonce=1e8b54d3-a601-499d-8444-3f629583de43
  Response Status Code: 404
  Response: {"error":"Tokens don't exist","statusCode":404}
[18328:18328:1012/184833.956443:VERBOSE7:refill_confirmation_tokens.cc(182)]   Headers:
    accept-ranges: bytes
    access-control-allow-credentials: true
    content-length: 47
    content-security-policy: sandbox
    content-type: application/json; charset=utf-8
    cross-origin-opener-policy: same-origin
    cross-origin-resource-policy: same-origin
    date: Thu, 12 Oct 2023 17:04:17 GMT
    fastly-restarts: 1
    referrer-policy: same-origin
    strict-transport-security: max-age=31536000; includeSubDomains; preload
    vary: Origin, Accept-Encoding
    via: 1.1 varnish
    x-cache: MISS
    x-cache-hits: 0
    x-content-type-options: nosniff
    x-frame-options: DENY
    x-research-status: 503
    x-served-by: cache-fra-eddf8230040-FRA
[18328:18328:1012/184833.958051:INFO:refill_confirmation_tokens.cc(188)] Failed to get signed tokens
[18328:18328:1012/184833.958235:VERBOSE1:account.cc(444)] Failed to refill confirmation tokens
[18328:18328:1012/184833.958475:VERBOSE1:account.cc(449)] Retry refilling confirmation tokens in 0 hours, 0 minutes, 26 seconds at 18:48:59.956
[18328:18328:1012/184834.650563:VERBOSE2:permission_rule_util.cc(16)] You do not have enough confirmation tokens
[18328:18328:1012/184834.650758:VERBOSE1:notification_ad_serving.cc(81)] Notification ad not served: Not allowed due to permission rules
[18328:18328:1012/184834.651299:VERBOSE1:user_idle_detection.cc(33)] User is active after 5 s
[18328:18328:1012/184839.284806:VERBOSE1:browser_manager.cc(56)] Browser did resign active
[18328:18328:1012/184839.286207:VERBOSE6:user_activity_manager.cc(41)] Triggered event: 17 (0:0:900 s)
[18328:18328:1012/184839.286694:VERBOSE1:browser_manager.cc(76)] Browser did enter background
[18328:18328:1012/184839.287607:VERBOSE6:user_activity_manager.cc(41)] Triggered event: 02 (0:0:900 s)
[18328:18328:1012/184859.967150:VERBOSE1:account.cc(453)] Retry refilling confirmation tokens
[18328:18328:1012/184859.967367:VERBOSE1:refill_confirmation_tokens.cc(166)] Get signed tokens
[18328:18328:1012/184859.967487:VERBOSE6:refill_confirmation_tokens.cc(170)] URL Request:
  URL: https://mywallet.ads.bravesoftware.com/v3/confirmation/token/5f1220cd-70f9-523c-a658-f5a0074bbb0b?nonce=1e8b54d3-a601-499d-8444-3f629583de43
  Method: kGet
[18328:18328:1012/184859.968098:VERBOSE7:refill_confirmation_tokens.cc(171)]   Headers:

[18328:18328:1012/184900.109224:VERBOSE6:refill_confirmation_tokens.cc(181)] URL Response:
  URL: https://mywallet.ads.bravesoftware.com/v3/confirmation/token/5f1220cd-70f9-523c-a658-f5a0074bbb0b?nonce=1e8b54d3-a601-499d-8444-3f629583de43
  Response Status Code: 200
  Response: {"batchProof":"VE8iXc6aJwcvIwvj6ZIxKGFaNBD19dQ8kG0YdPDPhQbnH+DxIc9uSmCRVfNff7r0nccXfntLWwdvPu0oya5yCg==","signedTokens":["tFEWQ0qum1c0LTX4RUFMDyLi21ZSq5ruoetxpJcZ9Vk=","7ujzA2lMj2bHwmGcxn+ndX0gt1BqqlrrsZsLtWVGNXU=","0rzqCqtX7sWRFBpqxXN3LPt4937ZAPdyMkE354i4z2o=","NsqA7xAkbl00qnEC3unTaKswTAhdoVwkSIYAPx4LgRw=","wL8oeMhREsaCVQhjSQwxVrVzqxcqfpxQ7irw0TYqK0Y=","rmapVVus34IQTiBJ0kBteh08Iu0aAUaUx05dl5Etdg8=","jlqOwbM/06jBBfVidgR4WHUscKr92VroU9WmeMlviFQ=","8mwWGji96rhL4VKGsXPXiP8CdGgwgQExk0BrvgOo60g=","hBEL+WyfaLlYJESxDKKBEhoLn04sPdupIm05rPPv8RE=","rIlqyVQFaey2kxLHaAwgbTnNxmu2saIjBlxYEiACvBw=","CKMlv+GmmFQvpnY3z8F98tN7yhj/y9MZsiwC5yu4LA0=","Pmi6suBrnRbzsfvnvwnz5sc3EEcPG0k7eQPzD31owWY=","bJiZT4I+FvHpe2UNCsCvFlVyujy36uLH3ShKf1MQeWQ=","LCUI6DfcLvOYM++zTPspTOc6DhNjmTfjB8K+2Nfnahs=","XoFQd77jOE2OqJvRSKxvZ5tyJclQqkgUU89DdblyLHg=","DgQfmgJMqj+KM6uxdLO2HlgK+2Jhs/QQcANchE7kNEU=","3NnQv6wrTzXMuH6/9Ba3Z3UHw52Z04uQN2f7t1R9Lhw=","3gF1j2RYExytsBUxoccSj4ptemIcfvhSrg/1dhzA8TQ=","MAH3rWAMZZLbVNK/urDEd5HD1cFdB5WMRU9+JugyHyM=","moafxjkkAkeXtrCa92b+igoIur2giKPavdeO3wERJGQ=","xmBM8fJZ6fMStIbiFXPyVVU/jTpQpHQFASTFcdBBdTk=","aDfhRcgE+8ZgWOJ6QNmHMQKeIhxLi8XlvcX/Rn0RCEg=","UntMADBAzwACyZpFa0hKEtPhImkR9+OPtaN7zGP0Ry0=","KHH4LVlWzGp4iq3Havv0VhaN4SVZ55CjPEesHdKCjww=","PhYA+5GyTd/XVRFIZOI17OaaguGkcASv8NWp+oBpQCI=","4AQbQd4XWOfVq+3tHZKKIK/cUN+y+QZMPwJHIVw/Bjk=","RkuWTa3ZvkP9/I9jDfgXvY5T+x2moAo/QPW7VeY/3zA=","IP5QWdlY8mNjhBz7wd5X1A5jzDgATxLzM0nZ4h5ug3s=","Pohy7CD2/weDNcq+Nfcjp+1qnbLnUucPKFty5Ux7x2k=","Fkaw2i0SeR0/gv+c7tPqaQp6O7VMIzoeJRsG0XQXnHs=","Gmk9leioyFzP+MbyJGmxtNuVPPgAdvftBdi8nFZEOF4=","OOSiWLJOanIrlQSZho6chQ5CibussPGlOc6Y3MfbUjE=","SlpdXjjOPwsHbTgn72to+EM1G7qfnMZcB2Y2dZOtzDo=","lo7jScgSaF3uxKeZZD4bqqaFHopaVH6/fbYX5PalKhc=","PsM2eYOwKQw0uKWWH9SYPknYHPQ8G8dm8WbIS7pG1HY=","wPOm0cO49KVmALsCPzRNpM/WyrpIgHdHzXI+SbPIiRk=","tHcfkJbeqgzCtdmVxyiYU2MgJs5nsO8MgHXOthsak1s=","CjA0yZr8gnm8LC/QvxzTJYIQJ8fbbLF02hdWAZ561X4=","7i1Mnwt3mUOQgDqhFLeh4eIMeUFR5Eirmpn66bvEf3Q=","9u7LaWCYqh0xVxSrlhF/Wd0pSSGkOoJL6xm4PplpNjk=","SKi1oCdk2PgmpcKUIJqTw6rT6rgptyt1kDDovCU5SRw=","WuPibAaZ/53CDGX+gELrWj53Lt74piIKrebT8Wxprgo=","/MOKeIJPT22NqSIbHLtBPez+kCfPCV/L30+C5iWOQX8=","hLeR+GryVKDLj2zQkVhDfCSqhMZOiOz+WMNCjYdTNkQ=","vNGVoH4pE3KGjJ2oengKqWZpxOwPBRQ5t5fPw7eBryM=","Ju3ALjMMDg0NwzxL2K3bfEtgyeaNt9pu1Jh4rqUqLXk=","KMMRYlApAj0Hcj0vV12DfaQcdIEaJ5+JBNQxunyuVww=","djhDY2BWYnHUhsFPd5IuSIB6U7gbAQL3vHFSST58pXE=","BL2qhOr0i+E9BgDi2QofPRMYopkpQSLDV6JoqOl+IHI=","hKEMgl7hl9oUqKFoXlkq+HD5mswcm3VIw64h8Vv4QV4="],"publicKey":"wgMhKwj/Vs832efOg+RlqJvLy3Ta3UDbz1wV+Z+dzEU="}
[18328:18328:1012/184900.110762:VERBOSE7:refill_confirmation_tokens.cc(182)]   Headers:
    accept-ranges: bytes
    access-control-allow-credentials: true
    content-length: 2531
    content-security-policy: sandbox
    content-type: application/json; charset=utf-8
    cross-origin-opener-policy: same-origin
    cross-origin-resource-policy: same-origin
    date: Thu, 12 Oct 2023 17:04:57 GMT
    fastly-restarts: 1
    referrer-policy: same-origin
    strict-transport-security: max-age=31536000; includeSubDomains; preload
    vary: Origin, Accept-Encoding
    via: 1.1 varnish
    x-cache: MISS
    x-cache-hits: 0
    x-content-type-options: nosniff
    x-frame-options: DENY
    x-research-status: 503
    x-served-by: cache-fra-eddf8230040-FRA
[18328:18328:1012/184900.123122:VERBOSE9:confirmation_state_manager.cc(180)] Saving confirmation state
[18328:18328:1012/184900.123123:VERBOSE1:get_signed_tokens_url_request_util.cc(160)] Added 50 confirmation tokens, you now have 50 confirmation tokens
```
